packages/now-client/src/upload.ts

```diff
@@ -5,6 +5,20 @@ import { fetch, API_FILES, createDebug } from './utils';
 import { DeploymentError } from '.';
 import deploy, { Options } from './deploy';
 
+const isClientNetworkError = (err: Error | DeploymentError) => {
+  if (err.message) {
+    // These are common network errors that may happen occasionally and we should retry if we encounter these
+    return (
+      err.message.includes('ETIMEDOUT') ||
+      err.message.includes('ECONNREFUSED') ||
```

Comment:
Probably add `ENOTFOUND` too

Reply:
Is there a list we could look at to make sure we catch all the scenarios?

Reply:
I'm not sure 🤔

Reply:
I'm not able to reproduce this though 🤔

Reply:
https://nodejs.org/api/errors.html#errors_common_system_errors

https://github.com/nodejs/node/blob/17e420b23f5462db9f1951d98233fdaee889c721/deps/npm/lib/utils/error-message.js#L292-L295

- ETIMEDOUT
- ECONNREFUSED
- ENOTFOUND
- ECONNRESET
- EAI_FAIL

Reply:
Nice! Added all of these 🙏🏻